projects/netgrif-components-core/src/lib/data-fields/i18n-field/abstract-i18n-errors.component.ts

```diff
@@ -15,8 +16,9 @@ export abstract class AbstractI18nErrorsComponent extends AbstractBaseDataFieldC
 
     protected constructor(protected languageIconsService: LanguageIconsService,
                           protected _translate: TranslateService,
-                          @Optional() @Inject(DATA_FIELD_PORTAL_DATA) dataFieldPortalData: DataFieldPortalData<I18nField>) {
-        super(dataFieldPortalData);
+                          @Optional() @Inject(DATA_FIELD_PORTAL_DATA) dataFieldPortalData: DataFieldPortalData<I18nField>,
+                          @Optional() @Inject(NAE_SAVE_DATA_INFORM) _saveDataInform: boolean | null = false) {
```

Comment:
I believe the default value (even if it doesn't work) should be provided only at the class that introduces the dependency (`AbstractBaseDataFieldComponent` in this case)